crates/bevy_log/src/lib.rs

```diff
@@ -166,7 +172,18 @@ impl Plugin for LogPlugin {
 
         let default_filter = { format!("{},{}", self.level, self.filter) };
         let filter_layer = EnvFilter::try_from_default_env()
-            .or_else(|_| EnvFilter::try_new(&default_filter))
+            .or_else(|from_env_error| {
+                if let Some(err) = from_env_error
+                    .source()
+                    .unwrap()
```

Comment:
is this safe to unwrap?

Reply:
Currently yes: https://docs.rs/tracing-subscriber/latest/src/tracing_subscriber/filter/env/mod.rs.html#809-814

But it might be more defensive to `and_then` it instead of unwrapping.

Reply:
Agreed, updated to use `and_then`. Thanks for mentioning it.